src/test/java/tech/jhipster/lite/generator/server/springboot/thymeleaf/domain/SpringBootThymeleafModuleFactoryTest.java

```diff
@@ -0,0 +1,32 @@
+package tech.jhipster.lite.generator.server.springboot.thymeleaf.domain;
+
+import static tech.jhipster.lite.module.infrastructure.secondary.JHipsterModulesAssertions.assertThatModuleWithFiles;
+import static tech.jhipster.lite.module.infrastructure.secondary.JHipsterModulesAssertions.pomFile;
+
+import org.junit.jupiter.api.Test;
+import tech.jhipster.lite.TestFileUtils;
+import tech.jhipster.lite.UnitTest;
+import tech.jhipster.lite.module.domain.JHipsterModule;
+import tech.jhipster.lite.module.domain.JHipsterModulesFixture;
+import tech.jhipster.lite.module.domain.properties.JHipsterModuleProperties;
+
+@UnitTest
+class SpringBootThymeleafModuleFactoryTest {
+
+  private static final SpringBootThymeleafModuleFactory factory = new SpringBootThymeleafModuleFactory();
+
+  @Test
+  void shouldCreateSpringBootThymeleafModule() {
+    JHipsterModuleProperties properties = JHipsterModulesFixture
+      .propertiesBuilder(TestFileUtils.tmpDirForTest())
+      .basePackage("com.jhipster.test")
+      .projectBaseName("myapp")
+      .build();
+
+    JHipsterModule module = factory.buildModule(properties);
+    assertThatModuleWithFiles(module, pomFile())
+      .hasFile("pom.xml")
+      .containing("<groupId>org.springframework.boot</groupId>")
```

Comment:
This is a weak assertion, it will always be satisfied with other spring boot dependencies.
You should rather verify that way:
```suggestion
      .containing(
        """
              <dependency>
                <groupId>org.springframework.boot</groupId>
                <artifactId>spring-boot-starter-thymeleaf</artifactId>
              </dependency>
        """
```